tests/integration/docusaurus/connecting_to_your_data/database/mysql_python_example.py

```diff
@@ -1,7 +1,8 @@
-from ruamel import yaml
-
 import great_expectations as gx
 from great_expectations.core.batch import BatchRequest, RuntimeBatchRequest
+from great_expectations.core.yaml_handler import YAMLHandler
```

Comment:
This eliminates some of the ruamel warnings

Reply:
Whoohoo! `YAMLHandler`!